docs/notebooks/downstream_machine_learning_regression_evaluation.ipynb

```diff
@@ -0,0 +1,311 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate synthetic data on downstream regression models\n",
+    "\n",
+    "### How to use this notebook\n",
+    "Use this notebook to understand the quality and performance of your synthetic or augmented data on downstream machine learning regression tasks. \n",
+    "\n",
+    "### Installation\n",
+    "Install Gretel Client to use Gretel's synthetic models as well as the Gretel Evaluate Regression model. You'll have to get your API key from the [Gretel console](https://www.console.gretel.ai) to configure your session. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -U gretel-client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gretel_client import configure_session\n",
+    "\n",
+    "configure_session(endpoint=\"https://api.gretel.cloud\", api_key=\"prompt\", cache=\"yes\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " ## Try: Generate synthetic data, then evaluate the synthetic data on regression models against real-world data\n",
+    " First, we'll generate synthetic data using a publicly available [processed Cleveland Clinic heart disease dataset](https://archive.ics.uci.edu/ml/datasets/heart+disease), which predicts the likelihood of heart disease present in a patient. We'll use Gretel's LSTM model to train on the real-world data and generate the synthetic data. For the regression evaluation, we'll predict the heart rate value in this target column: \"0\".\n",
```

Comment:
suggestion: Would be nice to add column names to this dataset that we have in AWS. There are somewhat more interpretable column names at https://archive.ics.uci.edu/ml/datasets/heart+disease. Something more descriptive than "0" would look a bit more polished I think. Even "target" or something generic be an improvement in my opinion. And giving some sort of name to the columns also makes the SQS report that will be available for the synthetic data a lot more understandable.

Reply:
that's a great suggestion! i'd like to open a separate ticket for adding descriptive names for all benchmark/gretel datasets missing headers - will have to cut out some time for this, there are a few missing

Reply:
we updated the datasets in benchmark, so this has been changed in notebook to be more descriptive